### DISCUSSION TITLE
feature idea possibly - locking accounts remotly by changing the txt in the dns value

### DISCUSSION THREAD

**Discussion by ghost** [bluesky-social/atproto#1815] [General Discussion]
for users- who verify with a custom domain -
 having the ability to remote lock a bluesky account by changing by changing the specific  text value: txt atproto value=locked  or value locked+emailaddress 

downside, if a users domain is expired - possible solutions prompt (additional security feature requires domain ownership to be in auto renew or expire in more than 2years to be able to enable feature. 

i know additional security features are on the road map, with mfa, or possibly yubiukeys, but for whatever situation happens, it could be a nice, additional step.  

priority low, but not sure if this would be possible.

**Comment by snarfed** [comment]
Hmm! What do you mean by "lock"?

**Reply by ghost** [reply]
> Hmm! What do you mean by "lock"?

Prevent any account access or activity, log anyone out. Any that here room for interpretation but anything to reduce harm if an account is compromised

**Reply by snarfed** [reply]
Interesting! Rotating signing key is probably the main way they've provided for that at the protocol level: https://atproto.com/guides/overview#account-portability . If you change your signing key to a new one that your PDS doesn't have, its commits to your repo will no longer be recognized by anyone else. There's no user-facing UX for that yet afaik though.
